validator/src/main.rs

```diff
@@ -1446,12 +1446,21 @@ pub fn main() {
             ..RuntimeConfig::default()
         },
         staked_nodes_overrides: staked_nodes_overrides.clone(),
-        replay_slots_concurrently: matches.is_present("replay_slots_concurrently"),
         use_snapshot_archives_at_startup: value_t_or_exit!(
             matches,
             use_snapshot_archives_at_startup::cli::NAME,
             UseSnapshotArchivesAtStartup
         ),
+        replay_forks_threads: if matches.is_present("replay_slots_concurrently") {
```

Comment:
might also move this logic into the new module. have a method that takes an `&ArgMatches` and returns a struct with all thread configs. destructure it above and use freely

Reply:
Done in https://github.com/anza-xyz/agave/pull/236/commits/1d438a1f5f44c5b5c1c8b91619ef15aaccad0eb3